### PR TITLE
Semantic Changes to CPDLC & YSBK Procedures

### PR DESCRIPTION
**Pull request by AgentWolfFBI**
## Summary
Includes some semantic changes to CPDLC for clarity, along with some minor formatting changes to YSBK SOPs.

I did re-order the vatACARs instructions to be more appropriate; controllers must add PDC {ICAO} / CPDLC {ICAO} to their remarks before attempting to connect to vatACARs (the program will not identify a station code unless this is in the remarks). Previous SOPs had connection to vatACARs before adjusting remarks.

## Changes
**Fixes**:
- Semantic changes to CPDLC for clarity
- Formatting changes to YSBK

**Changes**:
- Re-ordered vatACARs usage instructions to ensure controllers update remarks before attempting connection

### CODE REVIEW

File: docs/aerodromes/classd/Bankstown.md [hunk 1]
```diff
@@ -61,16 +61,16 @@ Circuits to be flown at `A010`
...
 ### General
-These procedures apply during hours of daylight only.  During hours of darkness, all helicopters must revert to fixed-wing operations.  
+These procedures apply during hours of daylight only. During hours of darkness, all helicopters must revert to fixed-wing operations.  
 
-The Main Pad (abeam taxiway Mike) is treated like a runway and requires a takeoff/landing clearance.  Helicopters are permitted to become airborne from a limited number of other locations on the aerodrome, such as taxiway November Two, and should be instructed to *"report airborne"* or *"report on the ground"*.
+The Main Pad (abeam taxiway Mike) is treated like a runway and requires a takeoff/landing clearance. Helicopters are permitted to become airborne from a limited number of other locations on the aerodrome, such as taxiway November Two, and should be instructed to *"report airborne"* or *"report on the ground"*.
 
 ### Reporting Points
-Three helicopter reporting points help keep helicopters segregated from other traffic.  
+Three helicopter reporting points help keep helicopters separated from other traffic.
```
Comment (alphadelta332) [inline docs/aerodromes/classd/Bankstown.md:69]: The original wording is correct. The reporting points do not automatically separate with other traffic, they *help* separate, by *segregating* the routes

File: docs/client/cpdlc.md [hunk 2]
```diff
@@ -4,36 +4,42 @@ title: CPDLC
 
 --8<-- "includes/abbreviations.md"
 
-In the real world, CPDLC operations are generally used by aircraft within airspaces in Class A airspace above `F245`.
+In the real world, CPDLC (Controller-Pilot Datalink Communications) is generally used by aircraft within Class A airspace above `F245`.
```
Comment (alphadelta332) [inline docs/client/cpdlc.md:7]: I have added a CPDLC abbreviation now here https://github.com/vatpac-technology/sops/pull/312 . So the expanded bracket won't be necessary

**Review by alphadelta332 — CHANGES_REQUESTED** [review]
Everything else looks good mate :)

File: docs/client/cpdlc.md (hunk 2 above)
Comment (AgentWolfFBI) [reply docs/client/cpdlc.md:7]: Excellent

File: docs/aerodromes/classd/Bankstown.md (hunk 1 above)
Comment (AgentWolfFBI) [reply docs/aerodromes/classd/Bankstown.md:69]: Gotcha, no worries

**Review by alphadelta332 — DISMISSED** [review]

**Review by LeGiraffe — DISMISSED** [review]

**Review by mattkelly4 — DISMISSED** [review]

**Review by tylerthetiletiler — DISMISSED** [review]

**Comment by tylerthetiletiler** [comment]
Hey can we get these conflicts fixed before we merge plz

**Comment by AgentWolfFBI** [comment]
> Hey can we get these conflicts fixed before we merge plz

Is this something I need to do

**Comment by tylerthetiletiler** [comment]
> > Hey can we get these conflicts fixed before we merge plz
> 
> Is this something I need to do

If you can... If not I can deal with it later

**Comment by AgentWolfFBI** [comment]
> > > Hey can we get these conflicts fixed before we merge plz
> > 
> > 
> > Is this something I need to do
> 
> If you can... If not I can deal with it later

Conflict resolved however I committed changes so re-review is req.

**Review by tylerthetiletiler — APPROVED** [review]

**Review by mattkelly4 — APPROVED** [review]

**Review by alphadelta332 — APPROVED** [review]

**Pull request merged**